client/components/EventsListContainer/index.jsx

```diff
@@ -75,7 +75,8 @@ class EventsListComponent extends React.Component {
                                 loadMoreEvents={this.props.loadMoreEvents}
                                 selectedEvents={this.props.selectedEvents}
                                 onEventSelectChange={this.props.onEventSelectChange}
-                                privileges={privileges} />
+                                privileges={privileges}
+                                session={session} />
```

Comment:
I would hold off on this idea a little longer because I'm concerned about 100s of event items subscribing to the store. Component creation or update is inevitable as their props change, but we should probably be a little more careful in making redux store connections.